examples/linear_model/plot_logistic_multinomial.py

```diff
@@ -50,7 +50,8 @@
     colors = "bry"
     for i, color in zip(clf.classes_, colors):
         idx = np.where(y == i)
-        plt.scatter(X[idx, 0], X[idx, 1], c=color, cmap=plt.cm.Paired)
+        plt.scatter(X[idx, 0], X[idx, 1], c=color, cmap=plt.cm.Paired,
```

Comment:
here I think the edgecolor is important to distinguish points from the background, so I think this is a good change.